invenio_app_ils/patrons/mappings/v6/patrons/patron-v1.0.0.json

```diff
@@ -4,14 +4,23 @@
       "date_detection": false,
       "numeric_detection": false,
       "properties": {
+        "$schema": {
+          "type": "keyword"
+        },
+        "email": {
+          "type": "text"
+        },
         "id": {
           "type": "integer"
         },
         "name": {
           "type": "text"
         },
-        "email": {
-          "type": "text"
+        "pid": {
```

Comment:
I noticed the https://dev-cds-ils.web.cern.ch/api/patrons/ 

```json
      {
        "created": null,
        "id": "1",
        "links": {
          "self": "https://dev-cds-ils.web.cern.ch/api/patrons/1"
        },
        "metadata": {
          "$schema": "patrons/patron-v1.0.0.json",
          "email": "patron1@test.ch",
          "id": "1",
          "location_pid": "enqhm-0n578",
          "name": "Yannic Vilma",
          "pid": "1"
        },
        "updated": null
      },
```

I am not sure when we introduced the `pid` as a patron property since patrons are not real records, and I see that the `id` is served as string, while it is define as integer.

Reply:
Nice one for the `id`, I will change it.
For the PID, we decided since the beginning to keep the PID for consistency: <https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/patrons/api.py#L80>